### DISCUSSION TITLE
How to change LineEdit text color?

### DISCUSSION THREAD

**Discussion by kalucky0** [slint-ui/slint#692] [Q&A]
I've tried looking throught SixtyFPS compiler code but there doesn't seem to be a way to change text color in LineEdit.
Will this be implemented?

**Comment by ogoffart** [comment]
We really need to add some way to style our Widgets.
This was also asked there https://github.com/sixtyfpsui/sixtyfps/discussions/683

Currently the only way to do that would be to re-implement LineEdit, which is not an easy component to re-implement.
